lib/data/monorepo.json

```diff
@@ -626,6 +626,11 @@
     "typefaces": "https://github.com/KyleAMathews/typefaces",
     "typescript-eslint": "https://github.com/typescript-eslint/typescript-eslint",
     "typography-js": "https://github.com/KyleAMathews/typography.js",
+    "ui5-web-components": [
```

Comment:
Not ui5-webcomponents like the repo names?

Reply:
Good catch, thanks for the feedback 👍